src/resmom/linux/alps.c

```diff
@@ -232,9 +269,12 @@ static element_handler_t handler[];
 #define EXPAT_BUFFER_LEN (65536)
 static char expatBuffer[(EXPAT_BUFFER_LEN * sizeof(char))];
 static char *basil_inventory;
+static char *basil_system_knl;
```

Comment:
This variable is not needed. I do not see any purpose of where it is being used

Reply:
Yes, you are right. I have made the changes.